src/wireviz/DataClasses.py

```diff
@@ -18,7 +18,7 @@
 Designator = PlainText  # Case insensitive unique name of connector or cable
 
 # Literal type aliases below are commented to avoid requiring python 3.8
-ConnectorMultiplier = PlainText  # = Literal['pincount', 'populated', 'unpopulated']
```

Comment:
`shorted`?

Reply:
Sorry, I was actually commenting the line below, where you added `shorts`. I was thinking that `shorted` is more consistent with the other options.

Reply:
Will remove it, since it is not used and not implemented, could be somthing that will be impleneterd later but one stap a the time.
This came from the trying to get the rigth numer for qty